grails-app/conf/UrlMappings.groovy

```diff
@@ -156,6 +156,13 @@ class UrlMappings {
             action = [POST:"exportCsv"]
         }
 
+        // Locations API
+
+        "/api/locations/getLocations"(parseRequest: true) {
```

Comment:
Can you explain the need for a new API endpoint? I'd prefer we use the existing API.